packages/manager/modules/freefax/package.json

```diff
@@ -26,7 +26,7 @@
     "@ovh-ux/ng-ovh-telecom-universe-components": "^5.0.0",
     "@ovh-ux/ng-ovh-ui-confirm-modal": "^2.0.0",
     "@ovh-ux/ng-ui-router-title": "^3.0.0",
-    "@ovh-ux/ui-kit": "^4.1.9",
+    "@ovh-ux/ui-kit": "^4.1.10",
```

Comment:
Technically this peer doesn't need an update because the module doesn't require change on the navbar. Also a patch ? Wasn't a feature added ?

Reply:
In first place, I've done it only for the navbar package. But it doesn't update at all to the needed version.
So I update it everywhere

Reply:
To update apps I usually run `yarn upgrade-interactive`, and then take care of the modules (for the peer dependencies)